website/docs/reference/widgets/modal.md

```diff
@@ -1,55 +1,93 @@
 # Modal
 
-Modal is a simple UI widget you can use when you want to create Dialogs, Popovers or Alerts.
 
-<VideoEmbed host="youtube" videoId="s8cHVkhj3ec" title="How to use Modal Widget" caption="How to use Modal Widget"/>
+This page explains how to use the Modal widget to create dialog windows in your app for displaying various types of content, such as alerts, confirmation pop-ups, forms, and more.
+
+
+<VideoEmbed host="youtube" videoId="s8cHVkhj3ec" title="Using the Modal widget" caption="Using the Modal widget"/>
+
+
+
+The Modal widget is a container used to group and handle related user inputs. You can open the Modal widget using actions, such as setting a Button widget's **onClick** event to open a Modal.
+
+:::note
+Modal widget remains hidden on the canvas and becomes visible only when an event is triggered. You can access and edit the Modal widget from the entity explorer. 
+:::
+
+## Create a modal
+
+When you drag a Modal widget, by default, the modal is pre-populated with Text and two Button widgets. These may be edited or removed as required. To customize the modal, you can add multiple widgets, such as text, images, forms, or other widgets.
```

Comment:
We discussed to remove this.  Instead explain the scenario when a user would want to use the modal to view and edit table row data.